highlight/vim/README.rst

```diff
@@ -0,0 +1,77 @@
+Chapel Mode for Vim
+===================
+
+Chapel support for Vim has been designed for use with Vim versions 5.08 or
+newer.
+
+
+File Descriptions:
+==================
+
++ **ftdetect/chpl.vim**
+
+  - Allows vim to auto-detect the syntax highlighting scheme for .chpl files.
+
++ **ftplugin/chpl.vim**
+
+  - Useful editor settings supporting chpl, chpldoc
+    
++ **syntax/chpl.vim**
+
+  - Syntax highlighting scheme for Chapel
+
++ **indent/chpl.vim**
+
+  - Indenting scheme for Chapel
+
++ **sample-vimrc**
+
+  - Sample .vimrc configuration file
+ 
+
+Installation Instructions:
+==========================
+
+- Create a directory in the root of your home directory called ".vim"
+
+  ``$ mkdir $HOME/.vim``
+
+- Copy the contents of this directory to your ".vim" directory
+
+  ``$ cp -r * $HOME/.vim``
+
+- Modify your .vimrc configuration file to enable automatic filetype
+  detection, syntax highlighting, and indenting (this step is optional but
+  recommended).
+
+  ``$ vim $HOME/.vimrc``
+
+- **OR:** Use the sample .vimrc file provided (be careful you don't overwrite
+  any existing .vimrc file!):
+
+  ``$ cp -i sample-vimrc .vimrc``
+
+
+Enabling Chapel Support in Vim:
+===============================
+
+If you have chosen to use the **sample-vimrc** file you can safely ignore this
+section.  Vim commands are denoted ": command".  You can give these commands
+manually whenever you start a vim session (with the ":") or they can also be
+given automatically every time vim is started by putting them in your .vimrc
+file (without the ":").
+
+- Enable automatic filetype detection:
+
+  ``: filetype on``
+
+- Enable syntax highlighting scheme:
+
+  ``: syntax on``
+
+- Enable indenting scheme:
+
+  ``: filetype indent on``
+
+..
+  vim:tw=78:ts=8:ft=help:
```

Comment:
I'm ok with dropping the modeline. vim-users with syntax highlighting should get `rst` syntax highlighting instead (which is the intended file format).